index/scorch/scorch.go

```diff
@@ -765,3 +787,53 @@ func parseToInteger(i interface{}) (int, error) {
 		return 0, fmt.Errorf("expects int or float64 value")
 	}
 }
+
+type fieldStats struct {
+	statMap map[string]map[string]uint64
+}
+
+func (fs *fieldStats) AddStat(statName, fieldName string, value uint64) {
```

Comment:
API name suggestions in https://github.com/blevesearch/scorch_segment_api/pull/38